Gruntfile.js

```diff
@@ -249,7 +256,7 @@ module.exports = function(grunt) {
     }
   });
 
-  grunt.registerTask('translate', ['validate', 'esbuild', 'add-locale']);
+  grunt.registerTask('translate', ['validate', 'esbuild', 'add-locale:newLang']);
```

Comment:
Shouldn't we generate this template every time we build?

Reply:
That happens here:
https://github.com/dequelabs/axe-core/pull/3536/files#diff-8e7e5f941c67b0cf481a9d650295119d4922b96117e7adc069ba30a81a78c65fR270

Because I added a `template` prop to `add-locale`, I need to explicitly call `add-locale:newLang`, otherwise both commands will run in `translate`.